web-client/src/presenter/sequences/serveDocumentErrorSequence.js

```diff
@@ -0,0 +1,9 @@
+import { clearModalStateAction } from '../actions/clearModalStateAction';
+import { setShowModalFactoryAction } from '../actions/setShowModalFactoryAction';
+import { unsetWaitingForResponseAction } from '../actions/unsetWaitingForResponseAction';
+
+export const serveDocumentErrorSequence = [
+  unsetWaitingForResponseAction,
+  clearModalStateAction,
+  setShowModalFactoryAction('WorkItemAlreadyCompletedModal'),
```

Comment:
💭 What would y'all think about naming the modal using a similar noun to the sequence? I found it a tad confusing that the sequence is speaking in terms of a `document` and then the modal that opens has to do with a `work item`.  Just a thought, let me know what you think!

Reply:
made a little change that gets around this fact--let me know if it works for ya.

Reply:
Thanks! I think I was referring more to renaming the sequence to say `workItemAlreadyCompletedErrorSequence` OR renaming the modal to be called `ServeDocumentErrorModal`, if that makes sense. I'm OK with merging this as-is, it was just a thought 😄 Thank you!